web/packages/design/src/assets/images/Assets.story.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from "react";
+
+import Image from "design/Image";
+import gravitationalLogo from "design/assets/images/gravitational-logo.svg";
+import kubeLogo from "design/assets/images/kube-logo.svg";
+import sampleLogoLong from "design/assets/images/sample-logo-long.svg";
+import sampleLogoSquire from "design/assets/images/sample-logo-squire.svg";
+import secKeyGraphic from "design/assets/images/sec-key-graphic.svg";
+import teleportLogo from "design/assets/images/teleport-logo.svg";
+import teleportMedallion from "design/assets/images/teleport-medallion.svg";
+import {TeleportLogoII} from "design/assets/images/TeleportLogoII";
```

Comment:
lets put new svg's here? https://github.com/gravitational/teleport/tree/600d14511cc263d87c72cd220995f4c1ce2d6699/web/packages/design/src/SVGIcon

Reply:
The only new SVG I added to `design/assets/images` is the TeleportLogoII. I added the story to view what was already in the directory.

I'm not sure it fits in /SVGIcon since /SVGIcon is icons, and this is a logo. The story for /SVGIcon assumes everything is an icon:
 
![Screenshot 2023-08-09 at 12 28 36 PM](https://github.com/gravitational/teleport/assets/11967646/92dc8a02-f7f0-4ecd-8ffa-50d678b828bd)

Thoughts?

Reply:
gotcha, we can leave it as is